app/middleware/error-handler.js

```diff
@@ -10,6 +10,10 @@ module.exports = function (err, req, res, next) {
     logger.info(`[${req.correlationId}] NotFoundError handled: ${err.message}. Rendering 404 page`)
     res.status(404)
     return response(req, res, '404')
+  } else if (err instanceof CouldNotCreatePaymentError) {
+    logger.info(`[${req.correlationId}] CouldNotCreatePaymentError handled: ${err.message}. Rendering 500 page`)
+    res.status(500)
+    return response(req, res, '500')
   }
```

Comment:
We probably don't need a specific error type for anything we'll render a 500 error page for.
We should probably handle any error that doesn't have a specific error type by rendering this page. 

This middleware currently calls `next` for any error that isn't handled here - what would we do with those errors? I think we should render the 500 page for any error we don't handle with a specific error page.

Reply:
Updated the middleware - `error-handler.js`
- For any generic errors - it now displays a `500` page.
- The existing `logger` object already sends a message to Sentry - so no need to add a separate Sentry log message.